css/portal-dashboard/all-responses-popup/popup-header.less

```diff
@@ -0,0 +1,235 @@
+@import "../variables";
+
+.popupHeader {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    height: @header-height;
+    background-color: @cc-teal;
+    color: white;
+
+    .icon {
+        width: 20px;
+        height: 20px;
+        fill: white;
+    }
+    &.inFeedbackMode {
+        background-color: @feedback-green;
+    }
+
+    .headerLeft {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: flex-start;
+        margin-left: 20px;
+
+        .assignmentIcon {
+            display: flex;
+            justify-content: center;
+            align-self: flex-start;
+            position: relative;
+            top: -1px
+        }
+        .title {
+            flex: 1;
+            width: 200px;
+            margin-left: 5px;
+        }
+    }
+    .toggleResponsesFeedback {
+        display: flex;
+        flex-direction: row;
+        flex: 2;
+        justify-content: center;
+        align-items: center;
+        height: 30px;
+        box-sizing: border-box;
```

Comment:
Add `cursor: pointer;` to get the correct cursor state when hovering the button.